src/sdk/nsfs_glacier_backend/backend.js

```diff
@@ -4,8 +4,14 @@
 const nb_native = require('../../util/nb_native');
 
 class GlacierBackend {
+    // These names start with the word 'timestamp' so as to assure
+    // that it acts like a 'namespace' for the these kind of files.
+    //
+    // It also helps in making sure that the persistent logger does not
+    // confuses these files with the WAL files.
```

Comment:
where is that filtering of timestamp files being done?

Reply:
In `PersistentLogger` `process_inactive` iterates over all the directory entries and filters the files based on the prefix. We have the a `PersistentLogger` named `migrate` so that instance selects all the files which start with `migrate...`

Reply:
to be honest this is a bit accidental. I would prefer a more clear separation of which files are logs and which files are not. why not use the suffix `.log` to filter them? I also think that a file that is named `timestamp.expiry` appears to have a suffix of `.expiry` which is a bit meaningless as a suffix, so I would rather have it with a `.timestamp` suffix still, and filter the logs based on their suffix. WDYT?

Reply:
I can see why it might be a little strange to have `expiry` as a suffix when looked as files. I actually just viewed them as key-value pairs, hence my usage of the term `namespace`.

Yeah, we can enforce that each log file must have the prefix `.log`. But that in itself won't be enough. We still need to filter out the files that are meant for **this** `PersistentLogger` instance. So maybe something like `<timestamp>.<type>.log` could be the format. The filtering could be for `<type>.log`. How does that sound?